hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/stream/RecordStreamFileWriter.java

```diff
@@ -249,18 +244,10 @@ public void closeCurrentAndSign() {
                     // write endRunningHash
                     final var endRunningHash = runningHash.getFutureHash().get();
                     recordStreamFileBuilder.setEndObjectRunningHash(toProto(endRunningHash.getValue()));
-                    dosMeta.write(endRunningHash.getValue());
                     LOG.debug(
```

Comment:
Do we know if modular code has similar logic ? I guess a question for Jasper. But just curious

Reply:
@Neeharika-Sompalli I need to investigate this actually; almost certain this exact issue won't exist, but do need to validate the replay/recovery scenarios.